en/docs/chapter_sorting/radix_sort.md

```diff
@@ -1,16 +1,16 @@
 # Radix sort
 
-The previous section introduced counting sort, which is suitable for scenarios where the data volume $n$ is large but the data range $m$ is small. Suppose we need to sort $n = 10^6$ student IDs, where each ID is an $8$-digit number. This means the data range $m = 10^8$ is very large, requiring a significant amount of memory space for counting sort, while radix sort can avoid this situation.
+The previous section introduced counting sort, which is suitable for scenarios where the data size $n$ is large but the data range $m$ is small. Suppose we need to sort $n = 10^6$ student IDs, where each ID is an $8$-digit number. This means the data range $m = 10^8$ is very large. Using counting sort in this case would require significant memory space. Radix sort can avoid this situation.
 
-<u>Radix sort</u> shares the core idea with counting sort, which also sorts by counting the frequency of elements. Building on this, radix sort utilizes the progressive relationship between the digits of numbers, sorting each digit in turn to achieve the final sorted order.
+<u>Radix sort</u> shares the same core concept as counting sort, which also sorts by counting the frequency of elements. However, radix sort builds upon this by utilizing the progressive relationship between the digits of numbers. It processes and sorts the digit one at a time, ultimately achieving the final sorted order.
```

Comment:
1. "However" -> "Meanwhile", we do not have a logical turn here, just making comparisons, so meanwhile is better.
2. "... and sorts the digit ...", it should be digits not digit.
3. "ultimately achieving the final sorted order" is a bit repetitive, with "ultimately", "achieving", "final", all have a meaning of at the end. "It processes and sorts the digits one at a time to achieve a sorted order" is better.